api/mod/tests/integration/test_users_router.py

```diff
@@ -13,8 +13,8 @@ def test_get_user_returns_correct_user(
     client: TestClient, db: Session, user: CreateUserRequest
 ) -> None:
     """Test GET /users/1"""
-    DBUser.create(db=db, user=user)
-    response = client.get("/users/1")
+    user = DBUser.create(db=db, user=user)
```

Comment:
I think user creation can be removed since it is done in the `db_engine` fixture now